apps/dh3-admin/src/index.html

```diff
@@ -6,7 +6,13 @@
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
```

Comment:
Please move this to the scss styles instead, so we don't have to do the import in every application:
@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600;700&display=swap');

Reply:
Question: If we import the font this way, wouldn't we lose the connection between the import and the `link rel="preconnect"` tags? It will look like they are there some sort of unnecessary legacy code. 

I don't think we can avoid not having some manual setup steps when generating a new application. For example, the `watt.scss` file still need to be added to the `styles` array in `angular.json` file for each new project in order to use the Design System. The `link rel="preconnect"` tags will need to be added manually as well.

I think we probably need a guide with all the steps necessary to properly setup a newly generated app.

Reply:
Due to performance we keep it as is, and document these in our usage / getting started